CME/dataobjects/CMEAccountEarnedCMECredit.php

```diff
@@ -6,8 +6,6 @@
  * @copyright 2011-2016 silverorange
  * @license   http://www.opensource.org/licenses/mit-license.html MIT License
  *
- * @property int        $id
- * @property ?SwatDate  $earned_date
```

Comment:
The `@var` where the property is defined should be made nullable too, I'd think.